NewProfile/index.ts

```diff
@@ -0,0 +1,45 @@
+import { AzureFunction, Context } from "@azure/functions";
+import { readableReport } from "@pagopa/ts-commons/lib/reporters";
+import { FiscalCode } from "@pagopa/ts-commons/lib/strings";
+import * as t from "io-ts";
+import { sha256 } from "../utils/conversions";
+
+const Failure = t.interface({
+  kind: t.keyof({
+    PERMANENT: null,
+    TRANSIENT: null
+  }),
+  reason: t.string
+});
+type Failure = t.TypeOf<typeof Failure>;
+
+const Success = t.interface({
+  kind: t.literal("SUCCESS")
+});
+type Success = t.TypeOf<typeof Success>;
+
+const permanentDecodeFailure = (errs: t.Errors): Failure =>
+  Failure.encode({
+    kind: "PERMANENT",
+    reason: `Cannot decode input: ${readableReport(errs)}`
+  });
+
+const index: AzureFunction = async (
+  context: Context,
+  inputFiscalCode: unknown
+) =>
+  FiscalCode.decode(inputFiscalCode)
+    .mapLeft(permanentDecodeFailure)
+    .bimap(
+      _ => {
+        context.log.error(`NewProfile|${_.reason}`);
+        return _;
+      },
```

Comment:
I think _ naming convention points to unused variable. Is it used here with a different meaning?

Reply:
Sometimes we just use `_` to mean "we don't need to name this thing, just pass it over"